Makefile

```
@@ -15,3 +20,5 @@ clean:
 	go clean -i ./...
 	rm -rf mysql-tester
 
+gen_perror:
```

```suggestion
build_gen_perror:
```

And then maybe something like this:
```
gen_perror: build_gen_perror
        ./gen_perror
```

---

Also `gen_perror` (the current one) should be added to `.PHONY:`